index.html

```diff
@@ -47,6 +48,157 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
     </ul>
   </section>
 
+<!-- works section -->
+
+  <section class="container">
+      <h1 class="working">My Recent <br>Works</h1>
+    <div class="card">
```

Comment:
:red_circle: :red_circle: [The project requiremen](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)t states that you should combine both Grid and Flexbox to lay out elements in this section . You used only flexbox. Kindly implement one of the project cards with CSS Grid  :arrow_down: 

![requir](https://user-images.githubusercontent.com/26736582/205437463-d7720e99-227b-4783-9550-481daa34a89c.png)